examples/inspection/plot_individual_conditional_expectation.py

```diff
@@ -0,0 +1,159 @@
+                                                    random_state=0)
+
+##############################################################################
+# Configuring a training pipeline
+# -------------------------------
+#
+# Let's configuring a training pipeline fit a MLPRegressor:
+
+print("Training MLPRegressor...")
+est = make_pipeline(QuantileTransformer(),
+                    MLPRegressor(hidden_layer_sizes=(50, 50),
+                                 learning_rate_init=0.01,
+                                 early_stopping=True))
+est.fit(X_train, y_train)
+print("Test R2 score: {:.2f}".format(est.score(X_test, y_test)))
+
+
+# We configured a pipeline to scale the numerical input features and tuned the
+# neural network size and learning rate to get a reasonable compromise between
+# training time and predictive performance on a test set.
+#
+# Note that it is important to check that the model is accurate enough on a
+# test set before plotting the ICE plots since there would be little use in
+# explaining the impact of a given feature on the prediction function of a
+# poor model.
+
+
+##############################################################################
+# ICE computation
+# ---------------
+#
+# Let's now compute the ICE plots for the neural network:
+#
+# One of the main limitation of ICE plots is that the plot get overcrowded
+# if many ICE curves are drawn. Due to this we will use a random sample of
+# 50 instances for the ICE plot.
+
+X_train_sample = X_train.sample(50, random_state=0)
+
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+
+print('Computing ICE plots...')
+plot_individual_conditional_expectation(est, X_train_sample, features,
+                                        n_jobs=3, grid_resolution=20,
+                                        centre=False, n_cols=2,
+                                        line_kw={'linewidth': 0.5})
+fig = plt.gcf()
+fig.suptitle('ICE of house value on non-location features')
+fig.subplots_adjust(hspace=0.3)
+
+
+# As the ICE curves do not have the same origin, sometimes it can be hard to
+# see the differences between ICE curves. The ICE curves can be made to have
+# same origin with ``centre`` parameter set to True.
+
+
+##############################################################################
+# Centered ICE computation
+# ------------------------
+#
+# Let's now compute the ICE plots with ``centre`` parameter set to True:
+
+print('Computing centered ICE plots...')
+plot_individual_conditional_expectation(est, X_train_sample, features,
+                                        n_jobs=3, grid_resolution=20,
+                                        centre=True, n_cols=2,
+                                        line_kw={'linewidth': 0.5})
+fig = plt.gcf()
+fig.suptitle('Centered ICE of house value on non-location features')
+fig.subplots_adjust(hspace=0.3)
+
+
+# In ICE plots it might not be easy to see the average effect of the 'target'
```

Comment:
This comment appears before the figure in the output (https://89544-843222-gh.circle-artifacts.com/0/doc/auto_examples/inspection/plot_individual_conditional_expectation.html#sphx-glr-auto-examples-inspection-plot-individual-conditional-expectation-py) and looks a bit out of place there.

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/0d9dafab8bd28297bb313e0eb701f101ea560786